### ISSUE TITLE
Error on vagrant up

### ISSUE THREAD

**Issue by aasanchez** [mysociety/fixmystreet#2101]
Error on Vagrant up
```
==> default: Running provisioner: shell...
    default: Running: inline script
    default: Checking submodules exist/up to date
    default: OpenSSL: error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version
    default: Unable to establish SSL connection.
    default: Can't locate local/lib.pm in @INC (@INC contains: /home/vagrant/fixmystreet/bin/../local/lib/perl5 /etc/perl /usr/local/lib/perl/5.14.2 /usr/local/share/perl/5.14.2 /usr/lib/perl5 /usr/share/perl5 /usr/lib/perl/5.14 /usr/share/perl/5.14 /usr/local/lib/site_perl .) at /home/vagrant/fixmystreet/bin/../setenv.pl line 20.
    default: BEGIN failed--compilation aborted at /home/vagrant/fixmystreet/bin/../setenv.pl line 20.
    default: Compilation failed in require at fixmystreet/bin/createsuperuser line 26.
    default: BEGIN failed--compilation aborted at fixmystreet/bin/createsuperuser line 27.
    default: ****************
```

**Comment by dracos** [comment]
I believe this was fixed in https://github.com/mysociety/fixmystreet/commit/274ca52df93495123e09a1f8b08dbe1411a2dc32 - could you check you are running the latest master? (You can check whether your Vagrantfile has precise64 or xenial64 in it.

**Comment by aasanchez** [comment]
Yes, I am using the last version... with all the changes, and happen to me using mac and linux (ubuntu)

**Comment by dracos** [comment]
What do you mean by 'last version'? We haven't released a version with that fix in, you need to be on master for it to work. What does `git status` and `git describe --tags` output?

Another option, do you have an out-of-date xenial64 box already on your system? I have 20180323.0.0.  The issue is (I think) GitHub now only allows TLS v1.2 connections, and older box versions might not have that support by default. `vagrant box update` should update it and then you can destroy/restart. Here's the start of my `vagrant up`, for example, to show that it does appear to be working okay here, at least, so something else must be going on:
```
§ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/xenial64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/xenial64' is up to date...
==> default: A newer version of the box 'ubuntu/xenial64' for provider 'virtualbox' is
==> default: available! You currently have version '20180323.0.0'. The latest is version
==> default: '20180420.0.0'. Run `vagrant box update` to update.
==> default: Setting the name of the VM: fixmystreet_default_1524415256129_61646
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 3000 (guest) => 3000 (host) (adapter 1)
    default: 3001 (guest) => 3001 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Connection reset. Retrying...
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default: 
    default: Guest Additions Version: 5.0.40
    default: VirtualBox Version: 5.1
==> default: Mounting shared folders...
    default: /home/vagrant/fixmystreet => /Users/matthew/Projects/mySociety/FixMyStreet/fixmystreet
==> default: Running provisioner: shell...
    default: Running: inline script
    default: Checking submodules exist/up to date
    default: 2018-04-22 16:41:34 URL:https://raw.githubusercontent.com/mysociety/commonlib/master/bin/install-site.sh [19928/19928] -> "install-site.sh" [1]
    default: DEVELOPMENT INSTALL
    default: ==== Installing fixmystreet
    default: Updating package lists... 
    default: done
    default: Installing some core packages...
    default:   git... 
    default: done
    default:   lockfile-progs... 
    default: done
    default:   curl... 
    default: done
    default:   dnsutils... 
    default: done
    default:   lsb-release... 
    default: done
    default: Testing for being on EC2... 
    default: done
    default: Testing 127.0.0.1.xip.io's IP address... 
    default: done
[...]
```

**Comment by aasanchez** [comment]
its workign
